metrics/qos/evm.go

```diff
@@ -0,0 +1,73 @@
+package qos
+
+import (
+	"fmt"
+
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/buildwithgrove/path/observation/qos"
+)
+
+const (
+	pathProcess            = "path"
+	evmRequestsTotalMetric = "evm_requests_total"
+)
+
+func init() {
+	prometheus.MustRegister(evmRequestsTotal)
+}
+
+var (
+	// evmRequestsTotal is a Counter metric for requests processed by an EVM QoS instance.
+	// It is incremented to track service requests and is labeled by:
+	// 	- `chain_id`: the ID of the chain using EVM as its QoS
+	//	- `request_method`: JSONRPC request's method field)
+	///     - `success`: whether the request was considered successfull, i.e. received a valid response.
+	// This is essential for monitoring EVM requests on different PATH instances.
+	//
+	// Usage:
+	// - Monitor EVM requests load.
+	// - Compare requests across different JSONRPC methods or chain IDs (i.e. different chains which use EVM as their QoS)
+	evmRequestsTotal = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Subsystem: pathProcess,
+			Name:      evmRequestsTotalMetric,
+			Help:      "Total number of requests processed by EVM QoS instance(s), labeled by chain ID and request method.",
+		},
+		[]string{"chain_id", "request_method", "success"},
```

Comment:
Why `sucess` and not `valid` like we use elsewhere? Ditto below

Reply:
`success` is used for the entirety of the request, whereas `valid` is attached to an endpoint response.
`valid` may later be added here separately, e.g. to indicate whether the user's request was valid.
I have added a TODO to capture this.